src/main/java/uk/gov/hmcts/reform/civil/validation/UnavailableDateValidator.java

```diff
@@ -149,4 +143,16 @@ private boolean isSmallClaimHearingNullOrEmpty(SmallClaimHearing smallClaimHeari
         return errors;
     }
 
+    public List<String> validateFuturePaymentDate(LocalDate paymentDate) {
```

Comment:
Please add test to existing test class for this method.

Reply:
test added